src/boot/efi/stub.c

```diff
@@ -912,16 +976,18 @@ static EFI_STATUS run(EFI_HANDLE image) {
         generate_embedded_initrds(loaded_image, sections, initrds);
         lookup_embedded_initrds(loaded_image, sections, initrds);
 
+        /* Measures ucode addons and populates all_initrds with the final order of all initrds */
+        measure_ucode_addons_and_create_final_list(initrds, &all_initrds, &n_all_initrds, ucode_addons, n_ucode_addons, &sections_measured);
```

Comment:
Is `sections_measured` the right one to use here?

Reply:
no. you are measuring into TPM2_PCR_KERNEL_CONFIG, hence us parameters_measured.